### PR TITLE
display up to five events per day

### PR DESCRIPTION
**Pull request by Haukinger**
added `IMultiEventDay` that allows a `DayEventCollection` to specify up to five individual colors for events that will be displayed as multiple dots on that day

grafik

### CODE REVIEW

**Comment by yurkinh** [comment]
Hi @Haukinger 
Thanks for this nice PR with the long-awaited feature that everybody asked for) 
But I never had time to implement this.
I will review and merge it ASAP.
Thanks a lot!

File: src/Plugin.Maui.Calendar/Shared/Controls/DayView.xaml [hunk 1]
```diff
@@ -13,36 +13,73 @@
...
                 Text="{Binding Date.Day, Mode=OneWay}"
                 TextColor="{Binding TextColor, Mode=OneWay}" />
-            <Border
-                Padding="0"
-                BackgroundColor="{Binding EventColor, Mode=OneWay}"
-                HeightRequest="8"
-                IsVisible="{Binding IsEventDotVisible, Mode=OneWay}"
-                StrokeShape="RoundRectangle 4"
-                WidthRequest="8" />
+            <StackLayout Orientation="Horizontal" HeightRequest="8">
```
Comment (yurkinh) [inline src/Plugin.Maui.Calendar/Shared/Controls/DayView.xaml:45]: We can utilize HorizontalStackLayout for better performance here

Comment (Haukinger) [reply src/Plugin.Maui.Calendar/Shared/Controls/DayView.xaml]: sure thing

**Comment by yurkinh** [comment]
Hi @Haukinger 
Could you please check your changes on iOS/MacOS?
Looks like Tap is not working on these platforms with your changes while in main branch everything looks fine

**Comment by Haukinger** [comment]
@yurkinh sorry for the delay, had to get hold of a mac :-) seems to work now on ios and android.

**Comment by mattyGazzera94** [comment]
so with the widget version 1.2.2 dated 3/09, is it possible to put up to a maximum of 5 dots per day in the monthly view?? Do I need any plugin updates?

**Comment by Haukinger** [comment]
@mattyGazzera94 this is not merged yet

**Comment by yurkinh** [comment]
Hi @mattyGazzera94 
I will try to recheck this PR and will publish it within the next few days. Also with updated maui nuget packages

**Comment by mattyGazzera94** [comment]
Thank you very much, as I have to release a version of my app in a few weeks and if it were possible to release it already with this functionality it would be optimal! I wait!

**Comment by Kiriakos41** [comment]
Why the first dot is always purple? can we fix it somehow? Is like purple green bown can somehow remove the first dot always is purple?

**Pull request merged**